geps/gep-1897/index.md

```diff
@@ -5,19 +5,16 @@
 
 ## TLDR
 
-This document specifically addresses the topic of conveying HTTPS from the Gateway
-dataplane to the backend (backend TLS termination), and intends to satisfy the single
-use case “As a client implementation of Gateway API, I need to know how to connect to
-a backend pod that has its own certificate”. TLS configuration can be a nebulous topic,
-so in order to drive resolution this GEP focuses only on this single piece of functionality.
+This document aims to address the use case of origination of TLS to backends from implementations.
+This is accomplished by a new BackendTLSPolicy API.
```

Comment:
I object to this obfuscation of the intent of BackendTLSPolicy.

Reply:
Everything cannot just be removed.  Please enumerate all the use cases, and keep this original use case, as described.

Reply:
Sorry I was doing some rewording/moving things around and didn't complete before I pushed out a WIP since it appeared there was heavy demand to have some signal of progress. Didnt expect everyone to actually start looking while it was in Draft mode! Definitely will have the use cases